Publish via CloudCannon (#284)
CloudCannon/platform-documentation: changelogs/2024-04-24_eleventy-3-support.mdx

---
_schema: default
title: Eleventy 3 support
type: minor
---

> This release provides support for automatically integrating sites using 11ty/Eleventy version 3.
> 
> ### Features & Improvements:
> 
> * Automatic integration plugin process expanded to support Eleventy upcoming version 3. All previously supported versions remain supported.

Reviewer: ```suggestion
This release added support for 11ty/Eleventy version 3.

### Features & Improvements:

* Expanded the automatic integration plugin process to support Eleventy version 3. All previous versions of Eleventy are still supported.
```
